<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>265</width>
    <height>115</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="font">
    <font>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="statusTip">
    <string/>
   </property>
   <property name="whatsThis">
    <string/>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,3,3">
        <item>
         <widget class="QLineEdit" name="equipment_change">
          <property name="text">
           <string>Выбор оборудования</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="type_equipment">
          <item>
           <property name="text">
            <string/>
           </property>
          </item>
          <item>
           <property name="text">
            <string>E-50-440-3,9ГМ</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="start_button">
          <property name="text">
           <string>Запуск оборудования</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout" stretch="0,10">
        <item>
         <widget class="QLineEdit" name="equipment_mode_2">
          <property name="text">
           <string>Выбор начальных условий</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="equipment_mode">
          <item>
           <property name="text">
            <string/>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Холодный резерв (котёл под водой)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Холодный резерв</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Горячий резерв</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Работа 20Т</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Работа 25Т</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Работа 30Т</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>265</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="setting"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="setting">
   <property name="text">
    <string>Настройки</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
